IDEAS/Fluid/Taps/BaseClasses/PartialTap.mo

```diff
@@ -0,0 +1,242 @@
+within IDEAS.Fluid.Taps.BaseClasses;
+partial model PartialTap "Partial model for a (DHW) tap"
+  extends IDEAS.Fluid.Interfaces.PartialTwoPortInterface(m_flow_nominal=0.2);
+
+  parameter Modelica.Units.SI.Temperature TSet=273.15+45
+    "Temperature setpoint of DHW at the tap";
+  parameter Modelica.Units.SI.Temperature TCol=273.15+10
+    "Temperature of cold supply water";
+
```

Comment:
Maybe directly write TSet=318.15 and TCol=283.15? Otherwise the default when opening the parameter window shows (273.15 + 45) - 273.15: (°C) and not 45 (°C) 
![image](https://github.com/open-ideas/IDEAS/assets/112491607/2cc34069-0c77-4ede-8681-5a129c82f26b)

Reply:
https://github.com/open-ideas/IDEAS/pull/1355/commits/ca79103891071632304cd2666becbe98f749a648